concrete/single_pages/dashboard/calendar/add.php

```diff
@@ -31,75 +31,92 @@
-    ?>
-            <input type="hidden" name="caID" value="<?=$calendar->getID()?>" />
-        <?php
-
-} ?>
-
-        <div class="form-group">
-            <?=$form->label('calendarName', t('Calendar Name'))?>
-            <?=$form->text('calendarName', $calendarName)?>
-        </div>
-
-        <div class="ccm-dashboard-form-actions-wrapper">
-            <div class="ccm-dashboard-form-actions ">
-                <a href="<?=$view->url($preferences->getPreferredViewPath())?>" class="btn btn-default pull-left"><?=t("Cancel")?></a>
-                <button type="submit" class="btn btn-primary pull-right"><?=$buttonText?></button>
-            </div>
-        </div>
-    </fieldset>
-    <fieldset>
-        <legend><?=t('More Details')?></legend>
-        <div class="form-group">
-            <?=$form->label('enableMoreDetails', t('Enable Additional Event Details'))?>
-            <?=$form->select('enableMoreDetails', [
-                '' => t('Do not link to a "More Details" page.'),
-                'create' => t('Create and link to a new "More Details" page.'),
-                'associate' => t('Link to an existing page')
-            ], $enableMoreDetails)
-            ?>
-        </div>
-
-        <div data-more-details-group="create" style="display: none">
-            <div class="form-group">
-                <?=$form->label('eventPageParentID', t('Parent Page'))?>
-                <?=Core::make('helper/form/page_selector')->selectPage('eventPageParentID', $eventPageParentID)?>
-            </div>
-
-            <div class="form-group">
-                <?=$form->label('eventPageTypeID', t('Page Type'))?>
-                <?=$form->select('eventPageTypeID', $types, $eventPageTypeID)?>
-            </div>
-
-            <div class="form-group">
-                <?=$form->label('eventPageAttributeKeyHandle', t('Calendar Event Attribute'))?>
-                <?=$form->select('eventPageAttributeKeyHandle', $attributeKeys, $eventPageAttributeKeyHandle)?>
-            </div>
-
-        </div>
-
-        <div data-more-details-group="associate" style="display: none">
-            <div class="form-group">
-                <?=$form->label('eventPageAssociatedID', t('Link to Page'))?>
-                <?=Core::make('helper/form/page_selector')->selectPage('eventPageAssociatedID', isset($eventPageAssociatedID) ? $eventPageAssociatedID : null)?>
-            </div>
-            <div class="help-block">
-                <?=t('<strong>Important</strong>: The page that you link to should contain a Calendar Event block or custom code that can render a specific calendar occurrence.')?>
-            </div>
-        </div>
-
-    </fieldset>
-</form>
+<div class="row">
+    <div class="col-9">
+        <section>
+            <p>
+                <b><?= t('Add a calendar using the form. ') ?></b>
+                <?=t('You may add multiple calendars, but you must add at least one before you can use the Calendar add-on')?>
```

Comment:
Can you add spaces to these and other `<?= ... ?>` to match our codestyle?

Reply:
Ooops, will fix that right a way 👍

Reply:
fixed :)